src/components/toolbar/index.js

```diff
@@ -117,13 +123,16 @@ class Toolbar extends Component {
               title={'Fullscreen'}
               icon={fullScreenIcon}
               tooltip={
-                fullscreen ? messages['fullscreen.off'] : messages['fullscreen.on']
+                fullscreen
+                  ? Localization['fullscreen.off']
+                  : Localization['fullscreen.on']
               }
               tooltipClass={'right-edge'}
               onClick={toggleFullscreen}
               disabled={renderError}
             />
           )}
+          <Select options={Object.keys(Localization._props)} />
```

Comment:
Not sure what is this for used :thinking:

Reply:
but I think it should use: `getAvailableLanguages()`

Reply:
Also probably just called once, not in update / render function.

Reply:
Sure... i am working on the changes.

Reply:
I have renamed all the keys according to their English equivalent value. Also thanks for your suggestion i didn't know about `getAvailableLanguages()`  earlier.